requirements.txt

```diff
@@ -1 +1,2 @@
-ops==2.0.0
\ No newline at end of file
+ops==2.0.0
+typer
```

Comment:
I am slightly confused here. I see in your pyproject.toml file that you have the following dependencies specified:

```toml
dependencies = [
    "ops>=2.0",
    "PyYAML==6.0"
]
```

but here you have the requirements set to `ops==2.0.0` and `typer`. Maybe requirements.txt should be set to:

```text
ops>=2.0
PyYAML==6.0
typer
```

and pyproject.toml should be set to:

```toml
dependencies = [
    "ops>=2.0",
    "PyYAML==6.0",
    "typer"
]
```

I guess `black` can just be an optional dependency for people who want their returned Python code to look pretty?